src/yoroi-wallets/storage.test.ts

```diff
@@ -0,0 +1,118 @@
+import AsyncStorage from '@react-native-async-storage/async-storage'
+
+import {storage as yoroiStorage} from './storage'
+
+describe('prefixed storage', () => {
+  beforeEach(() => AsyncStorage.clear())
+
+  it('getAllKeys, setItem, getItem, removeItem, clear', async () => {
+    const storage = yoroiStorage
+    await storage.getAllKeys().then((keys) => expect(keys).toEqual([]))
+
+    await storage.setItem('item1', item1)
+    await storage.getItem('item1').then((item) => expect(item).toEqual(item1))
+    await storage.setItem('item2', item2)
+    await storage.getItem('item2').then((item) => expect(item).toEqual(item2))
+    await storage.getItem('does not exist').then((item) => expect(item).toEqual(null))
+    await storage.getAllKeys().then((keys) => expect(keys).toEqual(['item1', 'item2']))
+
+    await storage.removeItem('item1')
+    await storage.getItem('item1').then((item) => expect(item).toEqual(null))
+    await storage.getAllKeys().then((keys) => expect(keys).toEqual(['item2']))
+
+    await storage.clear()
+    await storage.getAllKeys().then((keys) => expect(keys).toEqual([]))
+  })
+
+  it('getAllKeys, setItem, getItem, removeItem, clear', async () => {
```

Comment:
Maybe `getAllKeys, setItem, getItem, removeItem, clear with prefix`  so the text isn't the same as above